src/SocketTrebuchet.ts

```diff
@@ -45,6 +55,39 @@ class SocketTrebuchet extends Trebuchet {
     }, this.timeout * 1.5)
   }
 
+  private sendAck (mid: number) {
+    const ack = new Uint8Array(4)
```

Comment:
bumped it up to 4 bytes, ~2B messages before an overflow, which is fine.
note that if the server hits that limit, it's on the server to disconnect the client & reconnect (if we want to support that edge case)